src/Files.App/Helpers/FileOperationsHelpers.cs

```diff
@@ -914,7 +914,13 @@ public void WaitForCompletion()
 				operationsCompletedEvent.WaitOne();
 			}
 
-			protected override void Dispose(bool disposing)
+			public void Dispose()
+			{
+				Dispose(true);
+				GC.SuppressFinalize(this);
```

Comment:
I'd remove that method and put everyting that's in `Dispose(bool)` here

Reply:
@cinqmilleans